packages/bpk-component-theme-switcher/src/updateOnThemeChange-test.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import updateOnThemeChange from './updateOnThemeChange';
+import { THEME_CHANGE_EVENT, getHtmlElement } from './utils';
+
+const EnhancedComponent = updateOnThemeChange('div');
+
+describe('BpkThemeSwitcher', () => {
```

Comment:
Copypasta error

Reply:
I'll have to fix all these error in RTL as well!